spec/controllers/instructeurs/dossiers_controller_spec.rb

```diff
@@ -19,7 +19,7 @@
 
       it 'redirects to attestation pdf' do
         get :attestation, params: { procedure_id: procedure.id, dossier_id: dossier.id }
-        expect(response).to redirect_to(dossier.attestation.pdf_url.gsub('http://localhost:3000', ''))
+        expect(response.location).to match '/rails/active_storage/disk/'
```

Comment:
Est-ce que tu pourrais modifier le nom de la spec (le `it '…'`) pour préciser ce qu'on teste ?

Un truc du genre `it 'redirects to a persistent URL'` (si c'est bien le comportement qu'on veut)